.github/workflows/windows-ci.yml

```diff
@@ -49,8 +41,7 @@ jobs:
             -DPLASMA_VCPKG_NUGET_TOKEN="${{ secrets.GITHUB_TOKEN }}" `
             -DPLASMA_VCPKG_NUGET_RW=TRUE `
             ..
-          cmake --build . --config "${{ matrix.cfg.type }}" -j 2
-          cmake --install . --config "${{ matrix.cfg.type }}"
+          cmake --build . --target INSTALL --config "${{ matrix.cfg.type }}" -j 2
```

Comment:
I was curious if we need to pass `--config` in to each step (although I guess there's only one now), or if we could pass it in to the first cmake command and have it set for the remainder of them

Reply:
For multi-config generators (e.g. MSVC projects), I believe you do have to specify the config on each `--build` command...

Personal prefernce:  I would prefer to see the build and install targets kept separate, so they're easier to see in the build logs.

Also, we should probably run the `check` target here too; we may not have many tests, but we should try to keep them healthy.  Note that you'll need the Python DLL in PATH (or in the target's build directory) for the pfPythonTest to work.

Reply:
> Personal prefernce: I would prefer to see the build and install targets kept separate, so they're easier to see in the build logs.

Would you like them in a completely separate step in the YAML, or would you want separate calls `cmake --build .` for the `ALL_BUILD` and `INSTALL` targets?

> Also, we should probably run the check target here too; we may not have many tests, but we should try to keep them healthy. Note that you'll need the Python DLL in PATH (or in the target's build directory) for the pfPythonTest to work.

The check target is run as the last step in the YAML already. `VCPKG_APPLOCAL_DEPS` takes care of copying over the python DLL for us :smile:. I did it this way so we could get the installed artifacts, even if the tests fail.

Reply:
> 
> 
> Would you like them in a completely separate step in the YAML, or would you want separate calls `cmake --build .` for the `ALL_BUILD` and `INSTALL` targets in the current step?

Having configure, build, and install all as separate steps in the YAML makes the navigation nice in the build results, but it also adds a bit of extra complexity to the file...  I'd be fine with just separate cmake calls.

> 
> 
> The check target is run as the last step in the YAML already. `VCPKG_APPLOCAL_DEPS` takes care of copying over the python DLL for us 😄. I did it this way so we could get the installed artifacts, even if the tests fail.

Ah, I missed that, sorry (didn't expect it to be after "upload artifacts" ^_^)